Apps/ExampleDaemon/sdLitica.ExampleDaemonManagement/Program.cs

```diff
@@ -90,7 +90,10 @@ static void Main(string[] args)
                             series[i] = (double)rows[i].Fields["cpu"];
 
                         // invoke method (from F-sharp lib) given by OpName.
-                        log.Info(typeof(ExampleDaemonAnalysis.ExampleFunctions).GetMethod(@event.Operation.OpName).Invoke(null, new[] { series }));
+                        object operationResult = typeof(ExampleDaemonAnalysis.ExampleFunctions).GetMethod(@event.Operation.OpName).Invoke(null, new[] { series });
+                        log.Info(operationResult);
+                        string destinationPath = Path.Combine(Environment.CurrentDirectory, operation.Id.ToString() + ".txt");
+                        System.IO.File.WriteAllText(destinationPath, operationResult.ToString());
```

Comment:
File operation may ptentially cause exception, we need to fail gracefully in this case for now.

Reply:
this code is already inside of try-catch statement, so it will set error-status to operation. I added some logging to exception handling, we can expand exception-handling logic later